Documentation/RepoCompose.md

```diff
@@ -118,14 +118,14 @@ This command operates by providing json whose format is a subset of the output o
 
 A tool responsible for composing repos would ideally:
 
-1. Execute `run.cmd consumes` and capture the output.
-2. Alter the NuGet package versions in the json to have the correct build identifier: beta5, RTM, etc ..
-3. Execute `run.cmd change` and pass in the altered output.
+1. Execute `consumes` and capture the output.
```

Comment:
What kind of parameter are you imagining, what would it do? I think stdout is good because it's standard and doesn't require temporary file creation if we apply tools over the repo API to coordinate between repos.